app/src/main/java/ru/iteco/fmhandroid/entity/NurseStationEntity.kt

```diff
@@ -0,0 +1,27 @@
+package ru.iteco.fmhandroid.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import ru.iteco.fmhandroid.dto.NurseStation
+
+@Entity(tableName = "NurseStationEntity")
+data class NurseStationEntity(
+    @PrimaryKey
+    @ColumnInfo(name = "id")
+    val id: Int,
+    @ColumnInfo(name = "name")
+    val name: String,
+    @ColumnInfo(name = "blockId")
+    val blockId: Int,
+    @ColumnInfo(name = "comment")
+    val comment: String
+)
```

Comment:
А что если так? Типа декоратора запилить?
```suggestion
interface INurseStation {
    val id: Int
    val name: String
    val blockId: Int
    val comment: String
}

data class NurseStationEntity(
    @PrimaryKey
    @ColumnInfo(name = "id")
    override val id: Int,
    @ColumnInfo(name = "name")
    override val name: String,
    @ColumnInfo(name = "blockId")
    override val blockId: Int,
    @ColumnInfo(name = "comment")
    override val comment: String
) : INurseStation {
    
    data class WithEntity(
        @Embedded
        private val nurseStation: NurseStationEntity,
        @Relation(
            entity = RoomEntity::class,
            parentColumn = "id",
            entityColumn = "nurseStationId"
        )
        val rooms: List<RoomEntity>
    ) : INurseStation by nurseStation
}
```